### PR TITLE
[Fix-7844][docs] fix nnfriendliness and errors in the 2.x deployment documents

### PR DESCRIPTION
**Pull request by SunnyZ-L**
This closes [apache/dolphinscheduler issues 7844](https://github.com/apache/dolphinscheduler/issues/7844)

### CODE REVIEW

File: docs/en-us/dev/user_doc/guide/installation/pseudo-cluster.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ echo "dolphinscheduler" | passwd --stdin dolphinscheduler
 sed -i '$adolphinscheduler  ALL=(ALL)  NOPASSWD: NOPASSWD: ALL' /etc/sudoers
 sed -i 's/Defaults    requirett/#Defaults    requirett/g' /etc/sudoers
 
+# Unzip the downloaded DolphinScheduler binary package to the specified directory, such as creating /opt/dolphinscheduler, and rename the unzipped file name
+tar -zxvf apache-dolphinscheduler-2.0.3-bin.tar.gz -C /opt/dolphinscheduler
+mv apache-dolphinscheduler-2.0.3-bin  dolphinscheduler-bin
 # Modify directory permissions and grant permissions for user you created above
 chown -R dolphinscheduler:dolphinscheduler dolphinscheduler-bin
```
Comment (zhongjiajie) [inline docs/en-us/dev/user_doc/guide/installation/pseudo-cluster.md:42]: I think use in regexp mode is better, eg `chown -R dolphinscheduler:dolphinscheduler apache-dolphinscheduler-*-bin` WDYT

Comment (SunnyZ-L) [reply docs/en-us/dev/user_doc/guide/installation/pseudo-cluster.md:42]: OK，I'll change it

Comment (SunnyZ-L) [reply docs/en-us/dev/user_doc/guide/installation/pseudo-cluster.md]: > I think use in regexp mode is better, eg `chown -R dolphinscheduler:dolphinscheduler apache-dolphinscheduler-*-bin` WDYT

Modification complete

**Comment by zhongjiajie** [comment]
Well done, one more thing. we just release 2.0.3 document. Do you mind also changing 2.0.3 related doc? @SunnyZ-L

**Comment by SunnyZ-L** [comment]
> Well done, one more thing. we just release 2.0.3 document. Do you mind also changing 2.0.3 related doc? @SunnyZ-L

ok

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Pull request merged**